src/app/donation-start/donation-start-form/donation-start-form-parent.component.ts

```diff
@@ -14,43 +14,49 @@ import {
   PLATFORM_ID,
   ViewChild,
 } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
-import { MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
-import { MatCheckboxChange } from '@angular/material/checkbox';
-import { MatDialog } from '@angular/material/dialog';
-import { MatStepper } from '@angular/material/stepper';
-import { ActivatedRoute, Router } from '@angular/router';
-import { RecaptchaComponent } from 'ng-recaptcha';
-import { debounceTime, distinctUntilChanged, retryWhen, startWith, switchMap, tap  } from 'rxjs/operators';
-import { PaymentMethod, StripeCardElement, StripeElementChangeEvent, StripeError, StripePaymentRequestButtonElement } from '@stripe/stripe-js';
-import { EMPTY, Observer } from 'rxjs';
-
-import { AnalyticsService } from '../../analytics.service';
-import { Campaign } from '../../campaign.model';
-import { CampaignService } from '../../campaign.service';
-import { CardIconsService } from '../../card-icons.service';
-import { COUNTRIES } from '../../countries';
-import { Donation } from '../../donation.model';
-import { DonationCreatedResponse } from '../../donation-created-response.model';
-import { DonationService } from '../../donation.service';
-import { DonationStartMatchConfirmDialogComponent } from '../donation-start-match-confirm-dialog.component';
-import { DonationStartMatchingExpiredDialogComponent } from '../donation-start-matching-expired-dialog.component';
-import { DonationStartOfferReuseDialogComponent } from '../donation-start-offer-reuse-dialog.component';
-import { environment } from '../../../environments/environment';
-import { ExactCurrencyPipe } from '../../exact-currency.pipe';
-import { GiftAidAddress } from '../../gift-aid-address.model';
-import { GiftAidAddressSuggestion } from '../../gift-aid-address-suggestion.model';
-import { IdentityService } from '../../identity.service';
-import { ConversionTrackingService } from '../../conversionTracking.service';
-import { PageMetaService } from '../../page-meta.service';
-import { Person } from '../../person.model';
-import { PostcodeService } from '../../postcode.service';
-import { retryStrategy } from '../../observable-retry';
-import { StripeService } from '../../stripe.service';
-import { getCurrencyMaxValidator } from '../../validators/currency-max';
-import { getCurrencyMinValidator } from '../../validators/currency-min';
-import { EMAIL_REGEXP } from '../../validators/patterns';
-import { ValidateBillingPostCode } from '../../validators/validate-billing-post-code';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {MatAutocompleteSelectedEvent} from '@angular/material/autocomplete';
+import {MatCheckboxChange} from '@angular/material/checkbox';
+import {MatDialog} from '@angular/material/dialog';
+import {MatStepper} from '@angular/material/stepper';
+import {ActivatedRoute, Router} from '@angular/router';
```

Comment:
There are a few formatting changes in this file, most disappear if you select Hide whitespace in Github, the other is just adding line breaks in the imports from stripe-js. I think they're all fine to leave, probably were made automatically by PHP Storm. If the old version is better I can remove those changes and just keep the relevant stuff about countries.